xml/bp_troubleshooting_cephfs.xml

```diff
@@ -271,4 +271,382 @@ sudo zypper in ceph-common
      </para>
    </sect2>
  </sect1>
+ <sect1 xml:id="bp-troubleshooting-old-kernel-clients">
+   <title>Using legacy kernel clients</title>
+   <para>
+     Sometimes it is necessary to connect to a &cephfs; instance using old
+     kernel clients that may not support all the features required by the
+     cluster.  When this happens, the kernel client will fail to mount the
+     filesystem and will emit messages similar to the one shown below:
```

Comment:
This is ambiguous.

As I understand it, from a quick Google and bit of background reading, there is a Ceph client built into the Linux kernel. That should be specified and explained, at least briefly.

But here this talks about the _version_ of the kernel client.

That is confusing.

I presume that "client" here in general means "machines which are clients to the Ceph cluster." Is that correct?

I ask because I see 2 possible interpretations of "client" here: the Ceph client, versus the client computer.

So when we talk about versions, do we mean:

[1] The version of the Ceph client built into the kernel running on client machines?

(Is it possible that specific Linux distro/versions can have different versions of the Ceph client software? Because that is what this interpretation implies.)

[2] The version of the Linux kernel on the client machine(s)? 

(That is implied if the Ceph client version is tied to the version of the kernel.)

[3] The version of the Linux distribution running on client machines?

(Generally, assuming that the client machines' OS is up to date -- itself a **big** assumption -- then it seems to me that a specific version of the client OS will have a specific fixed version of the kernel, and if so, then to upgrade the Ceph client on the client machine means upgrading the kernel which means upgrading the distro version.)

Reply:
Right, I'm rephrasing the whole thing trying to clarify this.